client/trigger_dockerfile_lint.py

```diff
@@ -0,0 +1,34 @@
+#!/usr/bin/python
+
+import beanstalkc
+import json
+import os
+import sys
+
+if __name__ == "__main__":
+    name_space = sys.argv[1]
+    git_path = sys.argv[2]
+    filename = sys.argv[3]
+    notify_email = sys.argv[4]
+
+    # having '/' in a value used in os.path.join generates unexpected paths
+    if git_path.startswith("/"):
+        git_path = git_path[1:]
+
+    if filename.startswith("/"):
+        filename = filename[1:]
+
+    lint_job_data = {}
+
+    with open(os.path.join(os.environ['DOCKERFILE_DIR'], git_path, filename)) \
+    as f:
+        dockerfile = f.read()
+
+    bs = beanstalkc.Connection(host="openshift")
+    bs.use("start_linter")
```

Comment:
We should use `master_tube` here.
The flow should be, every trigger puts the job on `master_tube` with proper action, then `worker_dispatcher.py` reads every job coming over `master_tube` and routes the job based on action to respective tube (`start_linter`). 
So, the rule gonna be, the triggers / invokers put the job on master tube with proper action, `worker_dispatcher.py` acts as stop-relay point for routing the jobs. And all the receivers and readers reads the respective tubes (in your case `start_linter`), and do its job.

In this case, `worker_dispatcher.py` is bypassed, the job is put right from the trigger and received at worker. I dont think its correct.